node/comm/client.go

```diff
@@ -198,8 +198,21 @@ func Query(path string, packet []byte) interface{} {
 		return nil
 	}
 
-	var prototype interface{}
-	result, err := serial.Deserialize(response.Response.Value, prototype, serial.CLIENT)
+	var result interface{}
+
+	if path == "/applyValidators" ||
```

Comment:
The paths and handlers should probably be stored in a map so you can do something like:

```golang
	var path string
	paths := map[string]func(app.Application, map[string]interface{}) interface{}{
		"/applyValidators":     app.HandleApplyValidatorQuery,
		"/createExSendRequest": app.HandleCreateExSendRequest,
		"/createSendRequest":   app.HandleCreateSendRequest,
		"/createMintRequest":   app.HandleCreateMintRequest,
		"/createSwapRequest":   app.HandleSwapRequest,
		"/nodeName":            app.HandleNodeNameQuery,
		"/identity":            app.HandleIdentityQuery,
		"/signTransaction":     app.HandleSignTransaction,
		"/accountPublicKey":    app.HandleAccountPublicKeyQuery,
		"/accountKey":          app.HandleAccountKeyQuery,
		"/account":             app.HandleAccountQuery,
		"/validator":           app.HandleValidatorQuery,
		"/balance":             app.HandleBalanceQuery,
		"/version":             app.HandleVersionQuery,
		"/currencyAddress":     app.HandleCurrencyAddressQuery,
		"/sequenceNumber":      app.HandleSequenceNumberQuery,
		"/testScript":          app.HandleTestScript,
	}

	_, ok := paths[path]
	if ok {
		var proto interface{}
		result, err := serial.Deserialize(response.Response.Value, proto, serial.CLIENT)
	} else {
		err := clSerializer.Deserialize(response.Response.Value, &result)
	}
```